src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -168,13 +180,13 @@ private AIState completeBuild()
                 }
                 else
                 {
-                    if (wo instanceof WorkOrderBuildDecoration)
-                    {
-                        if (structureName.contains(WAYPOINT_STRING))
+                        if (wo instanceof WorkOrderBuildDecoration)
                         {
-                            worker.getColony().addWayPoint(wo.getBuildingLocation(), world.getBlockState(wo.getBuildingLocation()));
+                            if (structureName.contains(WAYPOINT_STRING))
+                            {
+                                worker.getColony().addWayPoint(wo.getBuildingLocation(), world.getBlockState(wo.getBuildingLocation()));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this line start at column 29. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)